go/fasthttprouter/go.mod

```diff
@@ -2,5 +2,5 @@ module main
 
 require (
 	github.com/buaazp/fasthttprouter v0.1.1
```

Comment:
I think it would be nice to create another version of this implementation, but using fasthttp/router
otherwise, looks good to me

Reply:
Sorry I'm not sure I understand @kirillDanshin 

This `PR` udate `fasthttp` but for 3 implementations :
+ #3 => `fasthttp` itself
+ #7 => `fasthttprouter`
+ #9 =>  `grouter` (when using `fashttp`)

All 3 are separated in https://github.com/the-benchmarker/web-frameworks#results

Reply:
@waghanza I was suggesting to to get back to fasthttp-related benchmarks later and add another one, using github.com/fasthttp/router, which is basically an updated and still maintained version of fasthttprouter :)

I have no actual fixes for this PR :)

Reply:
@kirillDanshin Sorry, I think I don't get you neither. Their is github.com/buaazp/fasthttprouter and github.com/fasthttp/router here. Do you suggest that one of those is not maintained anymore ?